test/unit/Github/Api/GraphQL/Query/GetMilestoneChangelog/Response/LabelTest.php

```diff
@@ -48,7 +48,7 @@ public function testInvalidPayload(array $payload): void
         Label::fromPayload($payload);
     }
 
-    /** @return iterable<int, list<array<string, mixed>>> */
+    /** @return iterable<int, array<array-key,array<string, mixed>>> */
     public function provideInvalidPayload(): iterable
```

Comment:
The reason why this was failing is that `list<>` contains an "empty" list type too.

As suggested, this can be iterable<int, array{array<string, mixed>}>`, which enforces the existence of the first array key inside each data provider entry :+1:

Reply:
Gotcha, I was not aware.